integration/registry/local-registry.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: registry-auth
+  namespace: ci
+data:
+  htpasswd: dGVzdHVzZXI6JDJ5JDA1JHVLUnAvdUV1aEMxRXpmbzQ1a1NCbk96eUlnV2tKNmk3R1VKTzNhLjVvY0JrZUNBNTZjRWxXCgo=
```

Comment:
Credentials for Docker registry

Reply:
Or another thought: I see below that you create the `registry-tls` secret as part of the script to setup the registry: why not also create this simple opaque secret there so that we only have the one location where the secret is defined and used? Just a thought, fine either way.

Reply:
`registry-tls` is created from the script as it needs some externally generated input (the key file). I prefer having everything possible put in a static yaml file, that is why `registry-auth` is there, it does not depend on anything.
But agree with your suggestion, I'll add a comment.